README.md

```diff
@@ -38,133 +38,23 @@ This layer supports the following security features:
   - U-Boot hardening.
   - FIT image signature checking (kernel, DTBs and ramdisk).
   - Rootfs signature checking via `dm-verity`.
+- Data-at-rest encryption
+  - Encryption key management via the Trusted Keys kernel subsystem
+  - Block device encryption with `dm-crypt`
 
-The features are currently supported on the following SoMs:
+For more information on the available features, please check the corresponding documentation:
 
-- Apalis iMX6
-- Apalis iMX8
-- Colibri iMX6DL
-- Colibri iMX6ULL (1GB eMMC variant only)
-- Colibri iMX7D (1GB eMMC variant only)
-- Colibri iMX8X
-- Verdin AM62
-- Verdin iMX8MM
-- Verdin iMX8MP
+| Documentation | Description |
+| :------------ | :---------- |
+| [docs/README-secure-boot.md](docs/README-secure-boot.md) | General documentation about the secure boot feature |
+| [docs/README-secure-boot-imx.md](docs/README-secure-boot-imx.md) | Details on the secure boot implementation for iMX based SoMs |
+| [docs/README-secure-boot-k3.md](docs/README-secure-boot-k3.md) | Details on the secure boot implementation for K3 based SoMs (e.g. AM62) |
```

Comment:
Suggestion (as done in the README-secure-boot document):

- "iMX based" -> "NXP iMX-based"
- "K3 based" -> "TI K3-based"

Reply:
Done!